### PR TITLE
PLNSRVCE-449: add kcp api files; various golang source checks

### PR DESCRIPTION
**Pull request by gabemontero**

### CODE REVIEW

File: config/kcp/apiresourceschema_jvmbuildservice.yaml [hunk 1]
```diff
@@ -0,0 +1,305 @@
+# This file is generated from CRDs by ./hack/generate-kcp-api.sh script.
+# Please do not modify!
+
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIResourceSchema
+metadata:
+  creationTimestamp: null
+  name: v202207261831.artifactbuilds.jvmbuildservice.io
```
Comment (gabemontero) [inline config/kcp/apiresourceschema_jvmbuildservice.yaml:8]: in comparing with the other repo's it dawned on me that we will want to get our group in line with the other controllers ... i.e. `appstudio.redhat.com`

I'd rather do that in a separate PR.

**Review by stuartwdouglas — APPROVED** [review]

**Review by Michkov — APPROVED** [review]
